base_multi_company/hooks.py

```diff
@@ -22,7 +22,7 @@ def set_security_rule(env, rule_ref):
         {
             "active": True,
             "domain_force": (
-                "['|', ('no_company_ids', '=', True), ('company_ids', "
+                "['|', ('company_ids', '=', False), ('company_ids', "
                 "'in', company_ids)]"
```

Comment:
You can improve it this way:
```suggestion
                "[('company_ids', 'in', [False] + company_ids)]"
```

Reply:
@pedrobaeza 
work done on the improvement